policyengine_canada/parameters/gov/provinces/mb/benefits/mbcb/reduction/two_children.yaml

```diff
@@ -0,0 +1,26 @@
+description: Manitoba Child Benefit phase out rate for families with two children.
```

Comment:
Manitoba phases the child benefit out for families with two children at this rate.